engine/lite/components/kafka-tests/src/test/scala/pl/touk/nussknacker/engine/lite/components/UniversalSourceJsonFunctionalTest.scala

```diff
@@ -45,15 +45,17 @@ class UniversalSourceJsonFunctionalTest extends AnyFunSuite with Matchers with S
   test("should test end to end kafka json data at sink / source") {
     val testData = Table(
       ("config", "result"),
-      //Primitive integer validations
+//      //Primitive integer validations
       // FIXME handle minimum > MIN_VALUE && maximum < MAX_VALUE) as an Integer to make better interoperability between json and avro?
       //      (sConfig(fromLong(Integer.MAX_VALUE.toLong + 1), longSchema, integerRangeSchema), invalidTypes("path 'Value' actual: 'Long' expected: 'Integer'")),
       (sConfig(fromInt(1), integerRangeSchema, longSchema), valid(fromInt(1))),
       (sConfig(fromLong(Integer.MAX_VALUE), integerRangeSchema, integerRangeSchema), valid(fromInt(Integer.MAX_VALUE))),
       (sConfig(obj(), objectSchema, objectSchema), valid(obj())),
       (oConfig(obj("first" -> fromString(""), "last" -> fromString("")), objectSchema, objectSchema, Input), oValid(obj("first" -> fromString(""), "last" -> fromString("")))),
-      (oConfig(InputEmptyObject, schemaObjString, schemaObjString, SpecialSpELElement("#input.field")), valid(obj())),
-      (oConfig(InputEmptyObject, schemaObjUnionNullString, schemaObjUnionNullString, SpecialSpELElement("#input.field")), valid(obj())),
+      (oConfig(Json.Null, schemaObjNull, schemaObjNull, Input), oValid(Json.Null)),
+      (oConfig(InputEmptyObject, schemaObjString, schemaObjString, Input), valid(obj())),
```

Comment:
shouldn't InputEmptyObject be the same as obj() ?

Reply:
InputEmptyObject and #input as output always returns empty object -> obj(). Please lookt at last changes.